### ISSUE TITLE
OpenBAFANG resets "Speed Meter Signals" to 1

### ISSUE THREAD

**Issue by FWfreiburg** [andrey-pr/OpenBafangTool#12]
First: the OpenBAFANG tool does not show th parameter "Speed Meter Signals" like the windows tools do.
Seond: using OpenBAFANG on my controller resets the parameter "Speed Meter Signals" to 1 which is needed to be set to 5 for correct speed recognition. Could you possibly add that parameter to your program? I would be verry Happy ;)

**Comment by andrey-pr** [comment]
Is this problem exists on uart or can system?

**Comment by andrey-pr** [comment]
Please send me log files

**Comment by FWfreiburg** [comment]
> Is this problem exists on uart or can system?

The problem is on uart. I will generate the logs asap but it may take some days. I will come back to you.

**Comment by andrey-pr** [comment]
> > Is this problem exists on uart or can system?
> 
> The problem is on uart. I will generate the logs asap but it may take some days. I will come back to you.

Just collect log file from it's directory:
```
on Linux: ~/. config/open-bafang-tool/logs

on macOS: ~/Library/Logs/open-bafang-tool/logs

on Windows: %USERPROFILE%\AppData\Roaming\open-bafang-tool/logs
```

**Comment by FWfreiburg** [comment]
sorry for the new issue. came from a handheld. (#13) here comes the log again.
[log.log](https://github.com/user-attachments/files/16489125/log.log)

**Comment by andrey-pr** [comment]
> sorry for the new issue. came from a handheld. (#13) here comes the log again.
> [log.log](https://github.com/user-attachments/files/16489125/log.log)
> 

Thank you. Please also send other log files from same directory (old.log or similar name, I don't remember)

**Comment by FWfreiburg** [comment]
It is the only file in ~/. config/open-bafang-tool/logs

**Comment by andrey-pr** [comment]
What is your version of software?

**Comment by FWfreiburg** [comment]
2.1.1-rc

**Comment by andrey-pr** [comment]
I already found reason why magnet number always sets to 1 (I forgot to remove from code temporary constant, that was added for test purposes). I will fix it in next release and add config to UI

**Comment by andrey-pr** [comment]
Hello. I just published new release 2.2.2 with parameter for number of magnets
